### PR TITLE
Tidy up repository!!

### LINKED ISSUE #53: Tidy up repository!!
Create 2 folders 

1. modules
Put these python module files in the modules folder. eg. "Image_To_array.py" , "label_binarizer_instance.py"

2. docs
Put these markdown files in the docs folder. eg. "transfer learning.md"
Extract "Installtion steps for tensorflow" from README.md and paste them in the docs folder as two separate md files for each OS

> Give links to these files in the Readme file.

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Issue that this pull request solves

Closes: #53  (issue number)

## Proposed changes
- Moved python module files in the modules folder
- Moved Markdown files in docs folder
- Updated Readme to include links of these files

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

@gshanbhag525 Please review it and dolet me know if any changes are required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,149 +1,18 @@
...
-pip install -U setuptools
-```
-
-#### <b>Step 5: Install TensorFlow</b>
-```
-pip install tensorflow
-```
-
-*For more details, visit the official tensorflow documentation page : https://www.tensorflow.org/*
+** Refer [this](https://github.com/CropAi/Data-Modeling/blob/master/docs/transfer learning.md) for knowing how to use transfer learning for Crop Disease Detection.
```
Comment (gshanbhag525) [inline README.md]: This link doesnt show up properly

Comment (gshanbhag525) [reply README.md]: Link to Transfer learning.md

Comment (Aman-Codes) [reply README.md]: @gshanbhag525 Sir the link will work only after you will merge this pull request as currently there doesn't exist any docs and module folder in your repository.

Comment (Aman-Codes) [reply README.md]: If you wish I could add the link of docs and modules folder of my repository for testing which you would have to change after merging the pull request.

Comment (gshanbhag525) [reply README.md]: @Aman-Codes No. My point was not of opening and viewing the link.
It was there is a whitespace in between transfer and learning.md 
So it disturbs the the link pattern and the link only gets tiil transfer. learning 
Screen Shot 2020-03-27 at 7 18 29 PM

Comment (Aman-Codes) [reply README.md]: @gshanbhag525 Thank you sir for pointing out that. Sir please guide me what should I do to rectify it shall I replace blankspace with %20 (hopefully that may work) or should I rename the transfer learning file to transfer_learning.md as in general it is not a good practice to add spaces while naming files.

Comment (gshanbhag525) [reply README.md]: renamin it to transfer_learning.md

Comment (Aman-Codes) [reply README.md]: @gshanbhag525 did the asked changes changes. Please review it and do let me know if any changes are required.

**Pull request merged**